### PR TITLE
chore: add github actions ci

### PR DESCRIPTION
**Pull request by iChenLei**
Add github actions ci cc @notslang

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
 {
   "name": "nib",
   "description": "Stylus mixins and utilities",
-  "version": "1.1.3",
```
Comment (notslang) [inline package.json:4]: bumping the version should be done in a separate tagged commit

Comment (iChenLei) [reply package.json]: @notslang resolved

**Comment by iChenLei** [comment]
@notslang Any update ? If you don't have enough time to maintian nib package, please add me as repository and npm collaborator
> my npm account is https://www.npmjs.com/~ichenlei

**Comment by iChenLei** [comment]
@tj holowaychuk sir, i need your help.

**Comment by iChenLei** [comment]
If you don't have enough time to maintian nib package, please add me as repository and npm collaborator. cc @notslang 
> my npm account is https://www.npmjs.com/~ichenlei

**Comment by iChenLei** [comment]
@notslang  squash to a single commit now, please review again. Thanks

**Comment by iChenLei** [comment]
Looking forward to new version v1.2.0 or v2.0.0 🤟  , Thank You ! @notslang

**Pull request merged**